diff --git a/node_modules/react-native-wheely/lib/WheelPicker.js b/node_modules/react-native-wheely/lib/WheelPicker.js
index 575325b..9eb241b 100644
--- a/node_modules/react-native-wheely/lib/WheelPicker.js
+++ b/node_modules/react-native-wheely/lib/WheelPicker.js
@@ -82,7 +82,7 @@ const WheelPicker = ({ selectedIndex, options, onChange, selectedIndicatorStyle
                     height: itemHeight,
                 },
             ] }),
-        react_1.default.createElement(react_native_1.Animated.FlatList, Object.assign({}, flatListProps, { ref: flatListRef, style: WheelPicker_styles_1.default.scrollView, showsVerticalScrollIndicator: false, onScroll: react_native_1.Animated.event([{ nativeEvent: { contentOffset: { y: scrollY } } }], { useNativeDriver: true }), onMomentumScrollEnd: handleMomentumScrollEnd, snapToOffsets: offsets, decelerationRate: decelerationRate, initialScrollIndex: selectedIndex, getItemLayout: (data, index) => ({
+        react_1.default.createElement(react_native_1.Animated.FlatList, Object.assign({}, flatListProps, { ref: flatListRef, style: WheelPicker_styles_1.default.scrollView, showsVerticalScrollIndicator: false, onScroll: react_native_1.Animated.event([{ nativeEvent: { contentOffset: { y: scrollY } } }], { useNativeDriver: true }), onMomentumScrollEnd: handleMomentumScrollEnd, snapToOffsets: offsets, decelerationRate: decelerationRate,  getItemLayout: (data, index) => ({
                 length: itemHeight,
                 offset: itemHeight * index,
                 index,
